src/App.tsx

```diff
@@ -1,61 +1,154 @@
+import { useState } from 'react';
+
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+import { PreparedTodo, Todo } from './types';
+
+function getUserById(userIdRecived: number) {
+  return (
+    usersFromServer.find((currentUser) => currentUser.id === userIdRecived)
+    || null
+  );
+}
```

Comment:
```suggestion
function getUserById(userId: number) {
  return usersFromServer.find(({ id }) => id === userId);
}
```

Reply:
I need whole `usersFromServer.find((user) => user.id === userId) || null;` cause user can not be found (i think, but i reimplement that little-bit)

Reply:
@VeDono By default, the `find` method returns `undefined` if it does not find anything, so it is not necessary to specify `null`, you can simply specify an additional type of `undefined` for the `user` interface
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find